static/docs/commands-reference/remote:modify.md

```diff
@@ -249,7 +250,7 @@ Now the config file should look like (run `cat .dvc/config`):
 
 ```ini
     ['remote "myremote"']
-    url = s3://mybucket/myproject
+    url = s3://path/to/dir
```

Comment:
why do we change this? on S3 it's usually a bucket + some path inside.

Reply:
I changed it to align with its previous command `dvc remote add -d myremote s3://path/to/dir` so that the url set earlier is reflected in the .dvc/config file as well. Should I change the previous command to `dvc remote add -d myremote s3://mybucket/myproject` or revert changes to its previous state?

Reply:
ah, yes I see. I would say it's a bug and the command above should be fixed instead. We should be using `s3://mybucket/storage` or something similar when we are dealing with S3.